test/local/verifier_upgrade_tests.js

```diff
@@ -75,7 +75,7 @@ createDBServer().then(
       .then(
         function (server) {
           var client
-          return Client.login(config.publicUrl, email, password, server.mailbox, {keys:true})
+          return Client.login(config.publicUrl, email, password, server.mailbox)
```

Comment:
I think this was an artifact of having sign-in confirmation enabled by default, which only gets triggered when requesting keys.